### ISSUE TITLE
Allow to restrict usage of wildcard imports

### ISSUE THREAD

**Issue by skuzzle** [skuzzle/restrict-imports-enforcer-rule#33]
As wildcard imports may lead to inaccurate results, we could implement a flag that will cause the analysis to fail if one is encountered.

Wildcard imports can be considered a bad practice, so this might be a helpful feature anyway

**Comment by skuzzle** [comment]
The special literal package part `'*'` can be used to match _only_ wildcarad imports:

Preventing a certain wildcarad import:
```xml
<bannedImport>java.util.'*'</bannedImport>
```
This pattern will prevent the usage of wildcard imports from java.util while still allowing explicit imports like `java.util.ArrayList`

Preventin all wildcard imports:
```xml
<bannedImport>**.'*'</bannedImport>
```

**Comment by skuzzle** [comment]
Will be released as 0.18.0
